definitions/checks/foreman_tasks/not_paused.rb

```diff
@@ -12,9 +12,17 @@ def run
       paused_tasks_count = feature(:foreman_tasks).paused_tasks_count(ignored_tasks)
```

Comment:
@upadhyeammit According to me we should also remove `ignored_tasks` parameter from here. Because even though we can ignore these tasks they should not be in the Paused state ever, especially while satellite upgrade. 
Let me know your thoughts on above.

Reply:
I agree, however I have these thoughts,

- This check is tagged as default which means it runs for health checks and at the time of upgrade checks and runs. So should we only skip ignored_task when we will actually do the upgrade check and upgrade run ?
- We do this check after restarting the services and as per my limited knowledge if we restart all services these ignored tasks should not be in paused state ? So here if we delete those while doing the health check they wont be created again ? Because after deletion we are not restarting services ? However at the time of upgrade, at very end of scenario we start the services.

What you think @patilsuraj767 and @kgaikwad ?

Reply:
Updates on above query:

Currently, this check is tagged as default so it is running as part of health as well as upgrade. 
As per offline discussion with @upadhyeammit & @patilsuraj767, it is worth verifying few things before any solution:
- What will be the impact when it deletes paused tasks including labels "Actions::Candlepin::ListenOnCandlepinEvents", "Actions::Katello::EventQueue::Monitor" because currently it is ignoring paused tasks with those labels?
- Would it be safe to run this deletion procedure when it won't perform service restart? 
Example - foreman-maintain health check command.
- Do we really need this check as part of health command? Other tasks related checks are tagged as "pre-upgrade"

Reply:
>     * What will be the impact when it deletes paused tasks including labels "Actions::Candlepin::ListenOnCandlepinEvents", "Actions::Katello::EventQueue::Monitor" because currently it is ignoring paused tasks with those labels?
> 

delete all of these, these tasks were removed from Foreman Tasks in 6.7 and are no longer processed as a task. Definitely delete them..

>     * Would it be safe to run this deletion procedure when it won't perform service restart?
>       Example - foreman-maintain health check command.

yes, any time you delete a paused task, you do not need to restart services.

> 
>     * Do we really need this check as part of health command? Other tasks related checks are tagged as "pre-upgrade"

I'd keep this with the health check as it is now as well as during the upgrade.